lab_2_retrieval_w_bm25/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "target_score": 0
+    "target_score": 4
```

Comment:
Where is start.py?  And what about 6, 8, 10?

Reply:
work in progress